studio.ps1

```diff
@@ -67,10 +67,24 @@ function Stage-Three {
     $kitArgs = "/q"
 
     if ($KITTYPE -eq "HLK") {
-        if (Test-Path -Path "$PSScriptRoot\Kits\HLK${HLKKITVER}\HLKSetup.exe") {
-            $kitPath = "$PSScriptRoot\Kits\HLK${HLKKITVER}\HLKSetup.exe"
-        } else {
-            $kitPath = "$PSScriptRoot\Kits\HLK${HLKKITVER}Setup.exe"
+        $mountedDisks = Get-WmiObject -Class Win32_LogicalDisk | Where-Object { $_.DriveType -eq 5 }

```

Comment:
- Merge `Where-Object { $_.DriveType -eq 5 }` into `foreach`.
- It says `mountedDisks`, but it never checks if mounted. Just name `disks`.